### PR TITLE
Clean up a few references to Firefox or Gecko (BL-12573)

### PR DESCRIPTION
**Pull request by andrew-polk**
This change is [Reviewable](https://reviewable.io/reviews/BloomBooks/BloomDesktop/6082)

### CODE REVIEW

**Review by StephenMcConnel — CHANGES_REQUESTED** [review]
Should this target Version5.5 which is webvew2 only?  If not, the only relevant comment may be the one about a change that doesn't seem to be related to Firefox/Geckofx.

> Reviewed 14 of 14 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/BloomBooks/BloomDesktop/6082#-:-Ne9Nni7-InoQUuOi6f2:boe2jef)* status: all files reviewed, 3 unresolved discussions (waiting on @andrew-polk)

___
*[`DistFiles/localization/en/Bloom.xlf` line 2877 at r1](https://reviewable.io/reviews/BloomBooks/BloomDesktop/6082#-Ne9MZ2YEQT62pw2Z_Hb:-Ne9MZ2YEQT62pw2Z_Hc:b3vlbv7) ([raw file](https://github.com/BloomBooks/BloomDesktop/blob/7f6cb7488b8f8122dd1b4e7db1b7979909f31f9c/DistFiles/localization/en/Bloom.xlf#L2877)):*
> ```XML
>         <note>ID: Errors.CannotConnectToBloomServer</note>
>         <note xml:lang="en">OLD TEXT (before 3.9, shorter wording): Bloom was unable to start its own HTTP listener that it uses to talk to its embedded Firefox browser. If this happens even if you just restarted your computer, ask someone to investigate if you have an aggressive firewall product installed. The agressive firewall product may need to be uninstalled before you can use Bloom.</note>
>         <note>Obsolete as of Bloom 5.6. (Replaced with Errors.CannotConnectToBloomServer.2 in BloomLowPriority)</note>
> ```

I think that should be Obsolete as of Bloom 5.5.  Even if it isn't.  :-)
___
*[`src/BloomExe/Program.cs` line 124 at r1](https://reviewable.io/reviews/BloomBooks/BloomDesktop/6082#-Ne9N2K7DXvczhzrKyCk:-Ne9N2K7DXvczhzrKyCl:b-nbankv) ([raw file](https://github.com/BloomBooks/BloomDesktop/blob/7f6cb7488b8f8122dd1b4e7db1b7979909f31f9c/src/BloomExe/Program.cs#L124)):*
> ```Smalltalk
> 			// Old comment: Firefox60 uses Gtk3, so we need to as well.  (BL-10469)
> 			// Aug 2023, we've moved away from GeckoFx/Firefox to wv2, but I don't know if this is still needed or not...
> 			GraphicsManager.GtkVersionInUse = GraphicsManager.GtkVersion.Gtk3;
> ```

I think it's still better to use Gtk3 instead of Gtk2, but I also think that may be Linux-only issue which doesn't matter in 5.5 and later.
___
*[`src/BloomExe/Publish/PublishView.Designer.cs` line 159 at r1](https://reviewable.io/reviews/BloomBooks/BloomDesktop/6082#-Ne9NStG43iOFXYiDXYn:-Ne9NStG43iOFXYiDXYo:b-n95om0) ([raw file](https://github.com/BloomBooks/BloomDesktop/blob/7f6cb7488b8f8122dd1b4e7db1b7979909f31f9c/src/BloomExe/Publish/PublishView.Designer.cs#L159)):*
> ```Smalltalk
>             this._L10NSharpExtender.SetLocalizableToolTip(this.exportAudioFiles1PerPageToolStripMenuItem, null);
>             this._L10NSharpExtender.SetLocalizationComment(this.exportAudioFiles1PerPageToolStripMenuItem, null);
>             this._L10NSharpExtender.SetLocalizationPriority(this.exportAudioFiles1PerPageToolStripMenuItem, L10NSharp.LocalizationPriority.NotLocalizable);
> ```

How does this relate to Firefox/Geckofx?

**Review by andrew-polk — COMMENTED** [review]
Changed target to 5.5.

> *[Reviewable](https://reviewable.io/reviews/BloomBooks/BloomDesktop/6082#-:-NeAL7CU8u83168OE5p6:b-vjh0hz)* status: 8 of 14 files reviewed, 3 unresolved discussions (waiting on @StephenMcConnel)

___
*[`DistFiles/localization/en/Bloom.xlf` line 2877 at r1](https://reviewable.io/reviews/BloomBooks/BloomDesktop/6082#-Ne9MZ2YEQT62pw2Z_Hb:-NeAKUte1hOtJc7bSos_:b-896fix) ([raw file](https://github.com/BloomBooks/BloomDesktop/blob/7f6cb7488b8f8122dd1b4e7db1b7979909f31f9c/DistFiles/localization/en/Bloom.xlf#L2877)):*
<details><summary><i>Previously, StephenMcConnel (Steve McConnel) wrote…</i></summary><blockquote>

I think that should be Obsolete as of Bloom 5.5.  Even if it isn't.  :-)
</blockquote></details>

Done.
___
*[`src/BloomExe/Program.cs` line 124 at r1](https://reviewable.io/reviews/BloomBooks/BloomDesktop/6082#-Ne9N2K7DXvczhzrKyCk:-NeAKWdF2XgMQPLVXvAb:b-896fix) ([raw file](https://github.com/BloomBooks/BloomDesktop/blob/7f6cb7488b8f8122dd1b4e7db1b7979909f31f9c/src/BloomExe/Program.cs#L124)):*
<details><summary><i>Previously, StephenMcConnel (Steve McConnel) wrote…</i></summary><blockquote>

I think it's still better to use Gtk3 instead of Gtk2, but I also think that may be Linux-only issue which doesn't matter in 5.5 and later.
</blockquote></details>

Done.
___
*[`src/BloomExe/Publish/PublishView.Designer.cs` line 159 at r1](https://reviewable.io/reviews/BloomBooks/BloomDesktop/6082#-Ne9NStG43iOFXYiDXYn:-NeAKY1s5-amxbQIl_7q:b-t4mhg0) ([raw file](https://github.com/BloomBooks/BloomDesktop/blob/7f6cb7488b8f8122dd1b4e7db1b7979909f31f9c/src/BloomExe/Publish/PublishView.Designer.cs#L159)):*
<details><summary><i>Previously, StephenMcConnel (Steve McConnel) wrote…</i></summary><blockquote>

How does this relate to Firefox/Geckofx?
</blockquote></details>

It doesn't, directly.
When I changed the message in `_openinBrowserMenuItem`, I figured it wasn't worth asking the translators to fix the localization. It is basically a debug menu thing, and I wasn't sure if it would survive the migration away from winForms, anyway.
So since we were no longer localizing the other message, I figured we might as well stop asking for new translations of this one, too.

Now, the downside is that existing translations will now go away.
So I would accept pushback if you think we should just keep it.

**Review by StephenMcConnel — APPROVED** [review]
> Reviewed 6 of 6 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/BloomBooks/BloomDesktop/6082)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @andrew-polk)
